src/functions.js

```diff
@@ -36,40 +42,59 @@ function cities( capitals, formatter ){
   // such as 'Paris is the capital of France'
 
   // apply formatter to each object in capitals array and return an array of resulting sentences
+return capitals.map(function(mm){return `${mm.city} is the capital of ${mm.country}`;})
 }
 
 function largerThanTen( numbers ){
   // numbers is an array of numbers
   // return a new array that contains only numbers from the input array which are greater than 10
   // hint: filter
+  return numbers.filter(function(number) { return number > 10;})
 }
 
 function even( numbers ){
   // numbers is an array of numbers
   // return a new array that contains only even numbers from the input array
   // hint: you may want to use the modulus operator '%'
+  return numbers.filter(function(numbe) { return !(numbe % 2);})
```

Comment:
This is a bit of a nitpick, but I would go with the explicitness of `return numbe % 2 === 0`